src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.setColor(color);
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a circle");
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String getFigureData() {
```

Comment:
The `getFigureData()` method concatenates strings, which is less efficient than using a `StringBuilder`. For better performance, especially in loops or when dealing with larger strings, consider refactoring this to use `StringBuilder`.